qcodes/__init__.py

```diff
@@ -53,7 +53,6 @@
     VisaInstrument,
     find_or_create_instrument,
 )
-from qcodes.instrument_drivers.test import test_instrument, test_instruments
```

Comment:
This should be noted as a breaking change with documentation about where to import it from. (not that I think this is very commonly used but still)

Reply:
@edumur In the future please comment directly on the thing that is unclear. That makes the conversation easier to follow. I mean in current qcodes the user can do `from qcodes import test_instrument` but with this change they will have to do 
`from qcodes.instrument_drivers.test import test_instrument` etc. 

We should document this

Reply:
Sorry for that!

My issue was more like where and how do you want me to do that.
I looked in the doc and currently there is not so much about theses functions: https://qcodes.github.io/Qcodes/search.html?q=test_instrument&check_keywords=yes&area=default.

Thanks!

Reply:
I just meant that we should have a breaking change note like the one that you added now. 

```
From now `test_intrument` and `test_intruments` must be imported from
`qcodes.instrument_drivers.test`.
```

I would perhaps just make it a bit more explicit. Like

```
The `test_intrument` and `test_intruments` functions are no longer available from the toplevel qcodes namespace. 
If you require these functions they must be imported from `qcodes.instrument_drivers.test`. 
```

or something like that